device/mellanox/x86_64-mlnx_msn4700-r0/Mellanox-SN4700-O28/buffers_defaults_objects.j2

```diff
@@ -92,8 +93,7 @@
     "BUFFER_PG": {
         "Dpc": {
             "active": {
-                "dynamic": "ingress_lossy_profile",
-                "static": "ingress_lossy_profile"
+                "dynamic": "NULL"
```

Comment:
@oleksandrivantsiv Could you check if WRED is applied on these internal ports? I believe thats also not required.

Reply:
@kperumalbfn let's keep WRED separate from this PR. If needed, we can create another one for WRED